conda_smithy/lint_recipe.py

```diff
@@ -71,11 +73,15 @@ def lintify(meta, recipe_dir=None):
             lints.append('The {} item is expected in the about section.'
                          ''.format(about_item))
 
-    # 3: The recipe should have some maintainers.
+    # 3a: The recipe should have some maintainers.
     if not extra_section.get('recipe-maintainers', []):
         lints.append('The recipe could do with some maintainers listed in '
                      'the `extra/recipe-maintainers` section.')
 
+    # 3b: Maintainers should be a list
```

Comment:
We should drop this numbering altogether. It only gets in the way when updating/adding more linting rules.